### PR TITLE
Update VISUAL_TESTING.md - eyes diff history drilldown

### PR DESCRIPTION
**Pull request by gilmeir**

### CODE REVIEW

File: docs/contribution/VISUAL_TESTING.md [hunk 1]
```diff
@@ -63,8 +63,17 @@ module.exports = {
 };
 ```
 
-## How can I debug it?
+## How can I debug a component rendered into a snapshot?
 Just run `npm run eyes-storybook-debug`. This will open the tests storybook and navigate between pages.
 
+## How can I debug the snapshots diff history?
+- Here's a [link](https://eyes.applitools.com/app/baselines/?accountId=mE99eP8CUn9UTSXSv98L3OSaJ0NZmAcY6Wndx6WSHODdY110&branch=wix%2Fwix-style-react%2Fmaster) to `master` branch in `wix-style-react`.
```
Comment (argshook) [inline docs/contribution/VISUAL_TESTING.md]: bear in mind this link is public but access is not public

Comment (gilmeir) [reply docs/contribution/VISUAL_TESTING.md]: I'm aiming here for maintainers.
Do you think this is not the right place?

Comment (gilmeir) [reply docs/contribution/VISUAL_TESTING.md]: @shlomitc do you think moving the eyes doc to wsr-issues will be more suitable?

Comment (gilmeir) [reply docs/contribution/VISUAL_TESTING.md]: Moved to a separate file and mentioned only Wix employees can access the dashboard.

**Pull request merged**